### ISSUE TITLE
Compiling bitlash for atmega64: /home/felicitus/sketchbook/libraries/bitlash/src/bitlash-parser.c:464:18: error: ‘GPIOR1’ was not declared in this scope

### ISSUE THREAD

**Issue by Drachenkaetzchen** [billroy/bitlash#11]
Hi,

when using bitlash with an atmega64, the compile process spits out the following error:

/home/felicitus/sketchbook/libraries/bitlash/src/bitlash-serial.c:401:7: error: ‘GPIOR0’ was not declared in this scope

The atmega64 doesn't have a GPIO register.

**Comment by billroy** [comment]
Yes, sorry, that is a platform-specific item and I don't have platform support for the Atmega64 yet.  (Pull requests welcome.)

You can change the definition to use another spare general-purpose register, or just use a byte variable, and it should work just fine.  Let me know if you need help sorting this out.

-br

**Comment by Drachenkaetzchen** [comment]
A simple workaround is to remove the define USE_GPIORS: https://github.com/billroy/bitlash/blob/master/src/bitlash.h#L692

**Comment by matthijskooijman** [comment]
It seems the relevant commit from #15 has already been merged, but didn't fix the issue completely. In commit 4321d4b in #23, I've really fixed this issue. @felicitus, could you check if this works for your Atmega64 setup (at least wrt the GPIO stuff)?
